### ISSUE TITLE
Error: Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by jurgob** [vkbansal/eslint-plugin-jsx-control-statements#12]
I get this lint error with the following code:

```
<For each="routeName" of={ Object.keys(routes) }>
  <div key={routeName} >
    <h2>{routeName}</h2>
       <For each="groupName" of={ Object.keys( routes[routeName] ) }>
         <RouteFieldsGroup key={groupName} title={groupName} fields={routes[routeName][groupName]} />
       </For>
    </div>
  </For>
```

The code itself work as expected, so it seems just a linter error:
below the error' s full details:

```
TypeError: Cannot read property 'value' of undefined
    at /Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint-plugin-jsx-control-statements/lib/rules/jsx-jcs-no-undef.js:61:90
    at Array.forEach (native)
    at EventEmitter.Program:exit (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint-plugin-jsx-control-statements/lib/rules/jsx-jcs-no-undef.js:39:33)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.leaveNode (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint/lib/util/node-event-generator.js:51:22)
    at CodePathAnalyzer.leaveNode (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:611:23)
    at CommentEventGenerator.leaveNode (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint/lib/util/comment-event-generator.js:111:23)
    at Controller.controller.traverse.leave (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint/lib/eslint.js:842:36)
    at Controller.__execute (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/jurgo.boemo/Workspace/platform/workdigital-dataplatform-admin-panel/src_js/search-admin/node_modules/eslint/node_modules/estraverse/estraverse.js:485:28)
```

this is my .eslint file:

```
{
  "ecmaFeatures": {
    "jsx": true,
    "modules": true
  },
  "env": {
    "browser": true,
    "node": true,
    "mocha": true,
    "jsx-control-statements/jsx-control-statements": true
  },
  "parser": "babel-eslint",
  "globals": {
    "expect": true,
    "browser": true
  },
  "rules": {
    "strict": [2, "never"],
    "indent": [2, 2, {"SwitchCase": 1}],
    "valid-typeof": 2,
    "block-scoped-var": 2,
    "eqeqeq": [2, "smart"],
    "no-var": 2,
    "no-caller": 2,
    "no-eq-null": 2,
    "no-invalid-this": 2,
    "no-labels": 2,
    "no-lone-blocks": 2,
    "no-loop-func": 2,
    "vars-on-top": 2,
    "brace-style": [2, "stroustrup", { "allowSingleLine": true }],
    "no-undef": 0,
    "jsx-control-statements/jsx-jcs-no-undef": 2,
    "no-unused-vars": [2, { "args": "after-used" }],
    "react/jsx-uses-react": 2,
    "react/jsx-indent": [2, 2],
    "react/jsx-uses-vars": 2,
    "react/jsx-no-undef": 2,
    "react/react-in-jsx-scope": 2
  },
  "plugins": [
    "react",
    "jsx-control-statements"
  ],
  "extends": ["plugin:jsx-control-statements/recommended"]
}
```

**Comment by AlexGilleran** [comment]
Looks like I forgot to do an undefined check :(

EDIT: Ah yeah seems you've nailed it in your PR :)

**Comment by vkbansal** [comment]
merged PR
